climakitae/util/utils.py

```diff
@@ -448,3 +451,68 @@ def hdh_cdh_lineplot(data):
     return data.hvplot.line(
         x="time", by="simulation", title=data.name, ylabel=data.name + " (degF)"
     )
+
+
+def convert_to_local_time(
+    data: xr.Dataset, selections: Select, lat: float, lon: float
+) -> xr.Dataset:
+    """
+    Converts the inputted data to the local time of the selection.
+    """
+    # 1. Find the other data
+    start, end = selections.time_slice
+    tz_selections = copy.copy(selections)
+    tz_selections.time_slice = (
```

Comment:
Just noticed this affects the Select Panel GUI so that it displays this time_slice...